### PR TITLE
Merge pull request #3 from Shreeshrii/master

### PR DESCRIPTION
**Pull request by Shreeshrii**

### CODE REVIEW

**Comment by zdenop** [comment]
@Shreeshrii : I do not see problem with links on https://tesseract-ocr.github.io/. Can you explain where/how did you see 404 errors?

**Comment by Shreeshrii** [comment]
I clicked on the links shown in Readme.md as shown in this repo,

**Comment by Shreeshrii** [comment]
The last link has full url and works, the others didn't.

http://tesseract-ocr.github.io/docs/

**Comment by zdenop** [comment]
ok. I see.

**Comment by Shreeshrii** [comment]
Similar changes also need to be made for all links from README in https://github.com/tesseract-ocr/tessdoc

@stweil Is there a way to link these correctly without using the full URLS?

File: README.md [hunk 1]
```diff
@@ -2,17 +2,17 @@
 
 ## Tesseract User Manual
 
-- [User Manual](tessdoc/)
+- [User Manual](http://tesseract-ocr.github.io/tessdoc/)
```
Comment (stweil) [inline README.md:5]: Please use https: instead of http: where possible.

**Comment by stweil** [comment]
> Is there a way to link these correctly without using the full URLS?

No. There are only two alternatives which both have advantages and disadvantages: use relative links which also allow local tests but don't work in the repository, or use absolute links which always point to tesseract-ocr.github.io and make local tests impossible. I preferred the first variant.

**Comment by Shreeshrii** [comment]
can any setting be tweaked to change this behaviour?

https://github.blog/2016-12-05-relative-links-for-github-pages/

File: README.md (hunk 1 above)
Comment (Shreeshrii) [reply README.md:5]: done via https://github.com/tesseract-ocr/tesseract-ocr.github.io/pull/4

**Comment by stweil** [comment]
Thank you for that link. I could fix the relative links for https://github.com/tesseract-ocr/tessdoc.

**Comment by Shreeshrii** [comment]
Thanks for fixing the links, @stweil.

I had also figured out same solution after some experimentation yesterday.

https://github.com/tesseract-ocr/tesseract/issues/2873#issuecomment-581006242

**Pull request merged**